### ISSUE TITLE
Time Triggered not working for me

### ISSUE THREAD

**Issue by edfxp** [eltrino/diamantedesk-application#42]
Hi,

Event Triggered works good but not for Time Triggered.

When created a new time triggered rule, which have every 5 mins to check if priority is medium then set ticket status to open, it doesn’t work in me. 

Do i need to set anything to make this work?  

Thanks
Ed

**Comment by edfxp** [comment]
I solved this issue by setting up a cron job, but i don't get any notification from Time Triggered rules when any properties get updated. Is this normal ?, because from Event triggered i got notification.

Thanks
Ed

**Comment by chris-freedom** [comment]
@edfxp it's okay, event triggered rules have predefine notify action that why you get emails, for time triggered rules you can set it manually in corresponding section.

**Comment by edfxp** [comment]
@chris-freedom How do i set the notification for this?, do i need to do coding or through admin setting? 
Thanks!

**Comment by edfxp** [comment]
I found this file /src/Diamante/AutomationBundle/Migrations/Data/ORM/CreatePredefinedRules.php.
After i create a new custom rule, how to compile it and make it works?, Thanks

**Comment by chris-freedom** [comment]
you don't need coding, just open you diamante admin and click on Rules -> Time Triggred menu item then press on "Create Time Triggered Rule" button and create custom rule for your needs.

**Comment by edfxp** [comment]
@chris-freedom Yes i did that and it worked, BUT it doesn't have any NOTIFICATION. How to get notification from time triggered rules?

**Comment by chris-freedom** [comment]
Can you show me screenshot of your rule?

**Comment by edfxp** [comment]
Sure, here it is
![screenshot-diamantedesk localhost-2017-03-21-23-45-58](https://cloud.githubusercontent.com/assets/26053394/24156180/d136faf2-0e90-11e7-9809-f9bd1c3eb5ff.png)

**Comment by edfxp** [comment]
@chris-freedom Did you find out the issue?

**Comment by chris-freedom** [comment]
@edfxp please check do you really have following command in your crontab 
`*/1 * * * * php /full/path/to/app/console diamante:cron > /dev/null`

**Comment by edfxp** [comment]
@chris-freedom Yes i did setup in cron and tried manually as well, because event triggered email notification works..

**Comment by chris-freedom** [comment]
@edfxp could you touch with us via email info at eltrino.com? We will try to help you directly.
